app/controllers/application_controller.rb

```diff
@@ -574,7 +574,7 @@ def pick_layout(*args)
   end
 
   def default_breadcrumb
-    label = "label_#{self.class.name.gsub('Controller', '').underscore.singularize}"
+    label = "label_#{self.class.name.demodulize.gsub('Controller', '').underscore.singularize}"
```

Comment:
I think `controller_name` is available here, so this can be replaced with:
```suggestion
    label = "label_#{controller_name.singularize}"
```